lib/-private/todo-handler.js

```diff
@@ -17,10 +17,10 @@ module.exports = class TodoHandler {
     this._processedResults = [];
   }
 
-  async update(filePath, results, todoConfig) {
+  update(filePath, results, todoConfig) {
     this._processedResults = this._buildResult(results);
 
-    await writeTodos(this._workingDir, this._processedResults, filePath, todoConfig);
+    return writeTodos(this._workingDir, this._processedResults, filePath, todoConfig);
```

Comment:
I'd prefer to keep this as an explicit async function. I realize that we are just returning a promise here, but declaring it as an async function makes it much easier to grok this file without being intimately familiar with the `@ember-template-lint/todo-utils` libraries API (e.g. does `writeTodos` return a promise?).

Reply:
After thinking on it, I think this might have been forced on you by linting configuration. If that is the case, we should absolutely change that config. Using an `async` without an `await` is perfectly fine/valid IMHO and we shouldn't prevent it.

Reply:
Correct. I can change the config.

Reply:
Done and this is fixed.